fcos-config.yaml

```diff
@@ -182,3 +182,19 @@ storage:
       contents:
         inline: |
           podman.crc.testing
+    - path: /etc/tmpfiles.d/podman-docker.conf
+      mode: 0644
+      contents:
+        inline: |
+          L+  /run/docker.sock   -    -    -     -   /run/podman/podman.sock
```

Comment:
What if we don't have the symlink, and we use `$ ssh -nNT -L $(pwd)/docker.sock:/run/podman/podman.sock -i /Users/prkumar/.ssh/podman-machine-default -p 64940 root@localhost`?

Reply:
@cfergeau No, it doesn't work.
```
<-- first terminal -->
➜  ssh -nNT -L /tmp/docker.sock:/var/run/podman.sock -p 2222 crc 
Warning: Permanently added '[127.0.0.1]:2222' (ED25519) to the list of known hosts.
channel 1: open failed: connect failed: open failed
channel 1: open failed: connect failed: open failed
channel 1: open failed: connect failed: open failed
channel 1: open failed: connect failed: open failed
channel 1: open failed: connect failed: open failed

<-- different terminal -->
➜  ~ export DOCKER_HOST=unix:///tmp/docker.sock
➜  ~ docker images                             
error during connect: Get "http://%2Ftmp%2Fdocker.sock/v1.24/images/json": EOF
```

Reply:
But with `/var/run/docker.sock` it works as expected.
```
<-- first terminal -->
➜ ssh -nNT -L /tmp/docker.sock:/var/run/docker.sock -p 2222 crc
Warning: Permanently added '[127.0.0.1]:2222' (ED25519) to the list of known hosts.

<-- different terminal -->
➜  ~ export DOCKER_HOST=unix:///tmp/docker.sock
➜  ~ docker images
REPOSITORY                        TAG                                        IMAGE ID       CREATED        SIZE
quay.io/crcont/gvisor-tap-vsock   3231aba53905468c22e394493a0debc1a6cc6392   f42b05e6b25e   5 months ago   6.65MB
```